packages/slate-html-serializer/src/index.js

```diff
@@ -121,6 +121,8 @@ class Html {
    */
 
   deserialize = (html, options = {}) => {
+    // Provides default value to make it working in IE 11.
```

Comment:
Could we update this comment to something along these lines: `Compat: Provide a default value for IE11 support`